content/en/docs/ops/configuration/extensibility/wasm-module-distribution/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Distributing WebAssembly Modules
+description: Describes how to make remote WebAssembly modules available in the mesh.
+weight: 10
+aliases:
+  - /help/ops/extensibility/distribute-remote-wasm-module
+  - /docs/ops/extensibility/distribute-remote-wasm-module
+keywords: [extensibility,Wasm,WebAssembly]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: n/a
+---
+
+Istio provides the ability to [extend proxy functionality using WebAssembly (Wasm)](/blog/2020/wasm-announce/).
+One of the key advantages of Wasm extensibility is that extensions can be loaded dynamically at runtime.
+But first these extensions must be distributed to the proxy.
+Istio makes this possible by allowing the Istio agent to dynamically download Wasm modules.
+
+## Configure an HTTP Filter with a Remote Wasm Module
```

Comment:
should we mention that this only works on `master` and 1.9?

Reply:
I suppose there could be some mention that this feature is new, not available in releases prior to 1.9. For example, on line 16: `Starting in version 1.9, Istio makes this possible by allowing ...`